README.md

```diff
@@ -56,105 +35,116 @@ Configuration is handled through environment variables, which are stored encrypt
-  * `./scripts/roombaht_ctl user 127.0.0.1 <env> wipe`
-* You can directly invoke a the django management tool, which gives you access to a variety of administrative tools.
-  * `./scripts/roombaht_ctl user 127.0.0.1 <env> manage shell` - invoke the django shell will full access to the orm and every module in the project.
-  * `./scripts/roombaht_ctl user 127.0.0.1 <env> manage user_show name@noop.com` the `user_show` command will display information on a guest.
-  * `./scripts/roombaht_ctl user 127.0.0.1 <env> manage user_edit --help` the `user_edit` command can edit limited aspects of a guest.
-  * `./scripts/roombaht_ctl user 127.0.0.1 <env> manage room_show --help` the `room_show` command will show details on a room.
-  * `./scripts/roombaht_ctl user 127.0.0.1 <env> manage room_edit --help` the `room_edit` command will allow editing of limited aspects of a room. There is no undo.
-  * `./scripts/roombaht_ctl user 127.0.0.1 <env> manage find_drama --help` the `find_drama` command will attempt to locate rooms and guests with known data corruption issues.
-  * `./scripts/roombaht_ctl user 127.0.0.1 <env> manage fix_room --help` the `fix_room` command will fix rooms with known data corruption issues. Pairs well with `find_drama`. There is no undo.
-    * `./scripts/roombaht_ctl user 127.0.0.1 <env> manage fix_room --help` the `fix_room` command will fix rooms with known data corruption issues. Pairs well with `find_drama`. There is no undo.
-    * `./scripts/roombaht_ctl user 127.0.0.1 <env> manage create_staff --help` the `create_staff` script replaces one half of the old `createStaffAndRooms.py` script. It will create and/also update staff entries.
-	* `./scripts/roombaht_ctl user 127.0.0.1 <env> manage create_rooms --help` the `create_rooms` script replaces one half of the old `createStaffAndRooms.py` script. It will create (and optionally - use caution and always `--dry-run` first) rooms. Make sure to specify `--hotel`. There is no undo other than a DB restore.
-
-# Data Population
+There are a variety of scripts used for managing either the production (`prod`) or staging/dev (`staging`) environments. Please contact an adult for information on SSH access, hostnames, and the location of a perfect dry martini. May of these commands are accessed via the `roombaht_ctl` script, which provides a commmon execution interface.
 
-*Note* This data population is meant to run only once after the initial DB migrations are applied. It has slightly different invocation for local dev and remote. This script will create the initial set of rooms and "staff" users. If the script detects that it has already been run on a database, it will ask for a confirmation. Because it does need to wipe the tables and start anew.
+```sh
+$ ./scripts/roombaht_ctl <user> <env> <command> <arg1> <arg2> ....
```

Comment:
Does env define which endpoint we want to look at?

Reply:
It extracts the `ROOMBAHT_HOST` var from the config which happens to also be our ssh endpoint.